ironfish/src/defaultNetworkDefinitions.ts

```diff
@@ -84,7 +84,8 @@ export const TESTNET = `{
       "genesisSupplyInIron": 42000000,
       "targetBlockTimeInSeconds": 60,
       "targetBucketTimeInSeconds": 10,
-      "maxBlockSizeBytes": 2000000
+      "maxBlockSizeBytes": 2000000,
+      "minFee": 1
```

Comment:
Let's set this to 0 so we don't hurt phase3 merging this is

Reply:
changed to 0